src/app/page.tsx

```
@@ -0,0 +1,236 @@
+                <Skeleton className="h-24 w-full" />
+                <Skeleton className="h-24 w-full" />
+              </div>
+            </div>
+          ) : error ? (
+            <div className="p-8 text-center bg-destructive/10 rounded-xl border border-destructive/20 shadow-sm">
+              <p className="text-destructive font-bold text-lg mb-1">{UI_LABELS[locale]?.errorTitle || UI_LABELS.en.errorTitle}</p>
+              <p className="text-sm text-muted-foreground">{error}</p>
+              <button
+                onClick={() => window.location.reload()}
+                className="mt-4 text-xs font-semibold underline underline-offset-4 hover:text-primary transition-colors"
+              >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)